### DISCUSSION TITLE
Welcome to Sunalyzer Discussions!

### DISCUSSION THREAD

**Discussion by BorisBrock** [BorisBrock/Sunalyzer#51] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BarryHavenga** [comment]
Hello
I have a Phocos 3 phase system.
If I install RS232 to USB connectors on each inverter, will I be able to pick up the 3 inputs in Sunalyzer?
